Behavior/BehaviorDelegate.swift

```diff
@@ -25,10 +25,12 @@ class BehaviorDelegate: SpeziAppDelegate {
                 AccountConfiguration(configuration: [
                     .requires(\.userId),
                     .requires(\.name),
+                    .requires(\.password),
+                    .requires(\.email),
```

Comment:
If you run the App in the Xcode simulator (with the play icon on the top left), you will see that the email field is actually showing up twice now. Once for the userId and the second time for the email requirement you specified.

For context, this is due to a small technical detail how the Firebase Account Service is implemented. It uses the user's email address as the user identifier so the email address is already collected via the userId (and actually if later on you query the email property of an existing account, it will fallback to returning the value of userId).
For reference: The [UserIdKey](https://swiftpackageindex.com/stanfordspezi/speziaccount/1.1.0/documentation/speziaccount/useridkey) documentation roughly touches on this behavior but it is not entirely clear about it. So this is definitely an area where we are gonna improve upon our documentation 🚀

If you just remove the line here, you are good to go 👍

Note: You might need to the reverse what is explained in [Get Started without Firebase](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/setup#Alternative-A-Get-Started-without-Firebase) to disable the --disableFirebase flag to get the Account Onboarding view showing up.